PyInstaller/compat.py

```diff
@@ -870,10 +870,13 @@ def check_requirements():
         if 'win32api' in sys.modules or 'pywintypes' in sys.modules:
             # Users should never see this error; if it occurs, it means someone
```

Comment:
I seem to remember trying this when I first made the changes, and it failed for some reason. I can't recall exactly why it failed, however.